linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml

```diff
@@ -24,6 +24,9 @@ severity: medium
 identifiers:
     cce@rhel8: 80934-3
 
+references:
+    srg: SRG-OS-000379-GPOS-00164
```

Comment:
Believe this should be a Nack. ``SRG-OS-000379-GPOS-00164`` states "The operating system must authenticate all endpoint devices before establishing a local, remote, and/or network connection using bidirectional authentication that is cryptographically based."

How does configuring bind crypto policies ensure the authentication of devices?

Reply:
So with this one my thought was that DNS transfers between two devices would require authentication.  

Chances are the named is not going to be installed or running (and /etc/named.conf it is not present in default installs) but this would force the proper configuration if it was installed.

Reply:
DNS server configuration falls under the DNS SRG. Any rules regarding DNS server should be removed from the RHEL STIG profiles.

DISA's new website doesn't seem to post the DNS SRG anymore. Copy at https://www.stigviewer.com/stig/domain_name_system_dns_security_requirements_guide/

Reply:
I agree.  Since the rule is in the OSPP Profile, will it be removed?

Reply:
Not sure. Brno just finished their rough draft of the OSPP profile. From here it will be reviewed by security specialists and NAPS.